docs/conf.py

```diff
@@ -0,0 +1,85 @@
+import os
+import sys
+
+import typing
+from typing import List
+
+typing.TYPE_CHECKING = True
+
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+sys.path.insert(0, os.path.abspath(".."))
+
+# -- Project information -----------------------------------------------------
+
+project = "snuba-sdk"
+copyright = "2021, Sentry Team and Contributors"
+author = "Sentry Team and Contributors"
+
+# The full version, including alpha/beta/rc tags
+release = "0.0.5"
```

Comment:
Is this supposed to be the version of the sdk? Can we have it centralized in a file and imported everywhere it is used?

Reply:
I think ultimately this is managed automatically. The code that generates the release also automatically updates the release number in the code base.